AmplifyPlugins/DataStore/Tests/DataStoreHostApp/AWSDataStorePluginAuthCognitoTests/AWSDataStoreCategoryPluginAuthIntegrationTests+Support.swift

```diff
@@ -11,16 +11,12 @@ import Amplify
 import DataStoreHostApp
 
 extension AWSDataStoreCategoryPluginAuthIntegrationTests {
-    func saveModel<T: Model>(_ model: T) {
-        let localSaveInvoked = expectation(description: "local model was saved")
-        Amplify.DataStore.save(model) { result in
-            switch result {
-            case .success(let todo):
-                localSaveInvoked.fulfill()
-            case .failure(let error):
-                XCTFail("Failed to save model \(error)")
-            }
+    func saveModel<T: Model>(_ model: T) async throws {
+        do {
+            let savedposts = try await Amplify.DataStore.save(model)
+            print("Local model was saved: \(savedposts)")
+        } catch {
+            XCTFail("Failed to save model \(error)")
         }
```

Comment:
let's apply the single expectation pattern here as well
```suggestion
        let localSaveInvoked = asyncExpectation(description: "local model was saved")
        Task {
            do {
                let savedposts = try await Amplify.DataStore.save(model)
                print("Local model was saved: \(savedposts)")
                await localSaveInvoked.fulfill()
            } catch {
                XCTFail("Failed to save model \(error)")
                throw error
            }
        }
        await waitForExpectations([localSaveInvoked], timeout: TestCommonConstants.networkTimeout)
```